drivers/video/video.c

```diff
@@ -3488,7 +3494,16 @@ int video_register(FAR const char *devpath,
 
 int video_unregister(FAR const char *devpath)
 {
-  return unregister_driver(devpath);
+  int ret;
+  struct stat buf;
+
+  ret = nx_stat(devpath, &buf, 1);
```

Comment:
unregister_driver will call nx_unlink internally, why do we need make this change?

Reply:
Ok, after compare the mainline with our local git, we found the similar issue when dealing with USB hotplug, and change unreigster_driver to fix the problem.
unreigster_driver is better place to fix this issue. @Donny9 could you upstream the patch? so @SPRESENSE could veriy whether your change work in this case too.

Reply:
@SPRESENSE please refs to https://github.com/apache/nuttx/pull/11892